extensions/wikia/ContentReview/ContentReview.hooks.php

```diff
@@ -341,26 +338,6 @@ public function onShowDiff( \DifferenceEngine $diff, &$notice ) {
 		return true;
 	}
 
-	public function onUserRightsGroupCheckboxes( $group, &$disabled, &$irreversible ) {
-		global $wgUser;
-
-		if ( $group === 'content-reviewer' && ( !$wgUser->isAllowed( 'content-review' ) || !$wgUser->isStaff() ) ) {
-			$disabled = true;
-		}
-
-		return true;
-	}
-
-	public function onUserAddGroup( \User $user, $group ) {
-		global $wgUser;
-
-		if ( $group === 'content-reviewer' && ( !$wgUser->isAllowed( 'content-review' ) || !$wgUser->isStaff() ) ) {
```

Comment:
So for this particular check, I talked to Adam and we agreed it's sufficient to specify that only members of content-reviewer group are allowed to assign this group (and skip the is staff check). So this is part of this definition:
https://github.com/Wikia/app/pull/9720/files#diff-8bf819f58bde706eaeeb9261498f35e5R384